pom.client.xml

```diff
@@ -333,6 +333,31 @@
               <outputDirectory>${project.reporting.outputDirectory}/test-coverage</outputDirectory>
             </configuration>
           </execution>
+          <execution>
```

Comment:
@JonathanGiles is this particular change going to require us to up the version and re-release the parent pom?

Reply:
@JimSuplizio since this change only introduces a check to fail the build when test coverage is low, it's not end user-facing. So, version update and re-release is not necessary.